src/loader/endpoints/GoogleDocsLib.py

```diff
@@ -0,0 +1,46 @@
+import pickle
+from googleapiclient.http import MediaIoBaseDownload
+from googleapiclient.discovery import build
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
+import io
+import pandas as pd
+
+# Takes the id and token and reads the bytes of a file
+def getFileBytes(file_id, token_path="token.pickle"):
+    token = pickle.load(open("token.pickle", "rb"))
+    drive_service = build("drive", "v3", credentials=token)
+    request = drive_service.files().get_media(fileId=file_id)
+    fh = io.BytesIO()
+    downloader = MediaIoBaseDownload(fh, request)
+    done = False
+    while done is False:
+        status, done = downloader.next_chunk()
+    return fh
+
+
+# Simply saves a google file to disk
```

Comment:
some descriptions are redundant. The name itself says what it is doing.

If an extra description is needed, then add it as a docstring:

def func():
"""
Description
"""